.editorconfig

```diff
@@ -0,0 +1,6 @@
+root=true
+
+[*.{groovy,java,kt,xml}]
+indent_style = tab
+indent_size = 4
```

Comment:
This project using `${PROJECT_ROOT}/etc/*.xml` to formatting code. There are formatters for IDEA and Eclipse, if you are using other editors, please let us know.

Reply:
From an engineering perspective, this is not a good practice. u can look any Povital project